### DISCUSSION TITLE
sense2vec extension attribute error

### DISCUSSION THREAD

**Discussion by jalal-git** [explosion/spaCy#6537] [Help: Coding & Implementations] [answered]
Hello guys! first of all let me thank you guys for your amazing library. man i trained a custom NER using spacy and it was crazy easy and clear. the other day i was reading about sense2vec and i likedit very much. so i downloaded the vector space (md) but altough im adding s2v to the pipe i still getting an error. check this out:

###code 
this part of my code is running flawlessly 
```
nlp = spacy.load("en_core_web_lg")
s2v = Sense2VecComponent(nlp.vocab).from_disk("s2v_reddit_2015_md/s2v_old")
nlp.add_pipe(s2v)
```

after writing this line:` nlp('Hello')._.s2v_vec`, im getting this error : `AttributeError: [E046] Can't retrieve unregistered extension attribute 's2v_vec'. Did you forget to call the `set_extension` method? `
 
how can i solve this problem? i followed the docs, so this error surprised me. and im sorry i know this is a noob question.

thanks! :)

**Comment by jalal-git** [comment]
I'm hopping to get the vector of "Hello" btw.

**Comment by svlandeg** [comment, marked as answer]
Ah, the error message is a little confusing here. The extension attribute isn't stored on the level of the `Doc`, but on the tokens and the spans, so you can do:
```
doc[0]._.s2v_vec
```

**Reply by jalal-git** [reply]
Thank you! this worked!
